src/plugins/python/pythonplugin.cpp

```diff
@@ -71,11 +71,18 @@ void PythonPlugin::initialize()
         Py_NoSiteFlag = 1;
         Py_NoUserSiteDirectory = 1;
 
+        #if PY_VERSION_HEX >= 0x03000000
```

Comment:
What is the point in still supporting older versions of Python here? It seems that the build files won't ever build against Python 2.7 and nor does `pythonbind.cpp` support older versions.

Reply:
I was actually wondering if we should bindings for both, but no, you're right, time to let 2 go